spec/poros/food_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+  it 'exists and has the correct attributes' do
+    # response = JSON.parse(File.read('spec/fixures/food_search_response.json'), symbolize_names: true)
```

Comment:
I appreciate that I can tell you considered both a fixture and an abridged attribute stubbed hash :)